src/internal/components/InternalThemeEditor.tsx

```diff
@@ -90,9 +132,8 @@ export const InternalThemeEditor = ({
           fields: () => (
             <ThemeSidebar
               themeConfig={themeConfig}
-              saveTheme={saveThemeSaveState}
               themeHistory={themeHistory!}
-              setThemeHistory={setThemeHistory}
+              handleThemeChange={handleThemeChange}
```

Comment:
onThemeChange as the prop - https://medium.com/@master_43681/understanding-and-proper-usage-of-on-and-handle-prefixes-in-react-function-naming-119f0beea400. It's really convention. I'm sure there are other places we should update too.